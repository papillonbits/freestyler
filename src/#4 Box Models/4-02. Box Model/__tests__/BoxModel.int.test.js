import { regular } from './BoxModel.int.story'

describe('<BoxModel />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
