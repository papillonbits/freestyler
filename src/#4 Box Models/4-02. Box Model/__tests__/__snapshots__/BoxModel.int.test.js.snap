// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BoxModel /> Render must match regular() 1`] = `
<div>
  <div
    class="box-model"
  >
    <h1>
      Box Model
    </h1>
    <div
      class="box"
    />
    <span
      id="outer-box"
    />
    <span
      class="label"
      id="outer"
    >
      Outer Box
    </span>
    <span
      class="label"
      id="margin"
    >
      Margin
       
    </span>
    <span
      class="label white"
      id="border"
    >
      Border
       
    </span>
    <span
      class="label"
      id="padding"
    >
      Padding
       
    </span>
    <span
      class="label"
      id="inner"
    >
      Inner Box
    </span>
    <span
      class="label"
      id="content"
    >
      <br />
      <br />
      ↑ 
      <br />
       Top
      <br />
       ← Left      Right →
      <br />
       Bottom 
      <br />
       ↓
       
    </span>
  </div>
</div>
`;
